src/unix/newlib/mod.rs

```diff
@@ -242,6 +219,57 @@ s! {
     }
 }
 
+cfg_if! {
+    if #[cfg(target_os = "horizon")] {
+        pub type sigset_t = ::c_ulong;
+
+        s! {
+            pub struct stat {
+                pub st_dev: ::dev_t,
+                pub st_ino: ::ino_t,
+                pub st_mode: ::mode_t,
+                pub st_nlink: ::nlink_t,
+                pub st_uid: ::uid_t,
+                pub st_gid: ::gid_t,
+                pub st_rdev: dev_t,
+                pub st_size: off_t,
+                pub st_atim: ::timespec,
```

Comment:
Are these time fields supposed to end in `tim` or `time`? I see compile errors with std using `tim`

Reply:
Oh, I see the type even changed, so I expect there's some std changes necessary?

Reply:
Good point, that's probably an oversight of mine. I can make a corresponding PR to rust-horizon tomorrow to address it.